src/main/java/com/mesabrook/ib/blocks/gui/telecom/GuiPhoneCrashed.java

```diff
@@ -20,7 +21,9 @@ public class GuiPhoneCrashed extends GuiPhoneBase
     private int delay = 0;
 
     MinedroidButton resetButton;
-    MinedroidButton copyStackToClipboardButton;
+    MinedroidButton uploadToPastebinButton;
+
+    ImageButton crashIcon;
```

Comment:
I feel like instead of using buttons we should probably just make a wrapper class that just draws images. Nothing you need to do here, just something I should probably add

Reply:
Yeah :/ the only alternative would be to draw the images directly, which is a pain in the ass lol. The image buttons are easier to use